packaging/docker/Dockerfile.debian

```diff
@@ -51,7 +51,10 @@ RUN apt-get update && \
 		liblua5.3-dev \
 		libhiredis-dev \
 		libnice-dev \
-		libmodbus-dev
+		libmodbus-dev \
+		python3 \
```

Comment:
Depending on `python3` here is not required as its a direct dependency of `python3-dev`.
See: https://debian.pkgs.org/12/debian-main-arm64/python3-dev_3.11.2-1+b1_arm64.deb.html

E.g. we are also not depending on `libmodbus`, and only on `libmodbus-dev`.